### ISSUE TITLE
Could not build for OSX on Travis 

### ISSUE THREAD

**Issue by ggtakec** [s3fs-fuse/s3fs-fuse#667]
#### Additional Information
Could not build for OSX(8.3) on travis CI now.

- Version of s3fs being used (s3fs --version)
 - https://github.com/s3fs-fuse/s3fs-fuse/tree/df3803c7b7c205b101a13b51678d5d4e6fb6fc05

#### Details about issue
Travis CI logs:
```
$ curl -sSL https://rvm.io/mpapis.asc | gpg2 --import -
/Users/travis/.travis/job_stages: line 57: gpg2: command not found
curl: (23) Failed writing body (0 != 2759)
```

**Comment by ggtakec** [comment]
The environment of Travis may have changed.
It seems that gpg2 can not be used.
Since s3fs does not depend on rvm, you should delete this execution line.

**Comment by ggtakec** [comment]
Fixed this by #668
